tests/sample/pictures/dir1/test2/index.md

```diff
@@ -1 +1,2 @@
-Order: 01
+Order: -10
+PartialOrder: 01
```

Comment:
The PartialOrder key doesn't seem to be used ?

Reply:
If you don't set that, then the PartialOrder test fails:

```
        settings["albums_sort_reverse"] = False
        a = Album("dir1", settings, album["subdirs"], album["medias"], gal)
        a.sort_subdirs(["meta.partialorder", "meta.order"])
>       assert [d.name for d in a.albums] == list(["test1", "test2", "test3"])
E       AssertionError: assert ['test2', 'test1', 'test3'] == ['test1', 'test2', 'test3']
E         At index 0 diff: 'test2' != 'test1'
E         Use -v to get more diff

tests/test_gallery.py:251: AssertionError
```

I suppose I could have reordered the test assertion, but it's a good check that it's using PartialOrder instead of Order.

Reply:
Oh right, I missed that ! 
Sounds good them, thanks again.